native/src/components/CalendarChoice.tsx

```diff
@@ -53,17 +48,22 @@ const CalendarChoice = ({
   const { t } = useTranslation('events')
   return (
     <Modal visible={overlayVisible} onRequestClose={closeOverlay} animationType='fade'>
-      <StyledView>
-        <TitleContainer>
-          <HeaderBackButton onPress={closeOverlay} label={t('back')} />
-          <Title>{t('chooseCalendar')}</Title>
-        </TitleContainer>
-        {calendars.map(cal => (
-          <CalendarButton key={cal.id} onPress={() => chooseCalendar(cal.id)} color={cal.color}>
-            <ButtonText>{cal.title}</ButtonText>
-          </CalendarButton>
-        ))}
-      </StyledView>
+      <Container>
+        <Header>
+          <HeaderBox goBack={closeOverlay} text={t('chooseCalendar')} textCentered />
+        </Header>
+        <ButtonContainer
+          contentContainerStyle={{
+            flex: 1,
+            justifyContent: 'center',
```

Comment:
Can't we use styled components for this as well? Best keep it consistent

Reply:
Apparently, we can! Just need the `attrs` in there, thanks for the hint!